File: java/1328A.java
// 1328 A. Divisibility Problem

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();

        while (n > 0) {
            n--;
            int a = scanner.nextInt();
            int b = scanner.nextInt();
            int result = 0;

            if (a % b == 0) {
                System.out.println(result);
                continue;
            } else {
                result = b - (a % b);
            }

            System.out.println(result);
        }
    }
}

File: cpp/1328A.cpp
// 1328 A. Divisibility Problem

#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    while(n--){
        int a,b,result = 0;
        cin >> a >> b;

        if(a % b == 0){
            cout << result << endl; 
            continue;
        }
        else {
            result = b - (a % b);
        }

        cout << result << endl;
    }
    return 0;
}
